### PR TITLE
BIP 32: Private key derivation operation order fix

### PR DESCRIPTION
**Pull request by gspasov**
Added additional parentesis for clear understanding of the equation on this row. I am doing this proposition because I was stuck on this part for quite some time untill I understood that I furst need to add and then do the modular devision. I think that this way, the order of operations is more clear.

### CODE REVIEW

**Comment by luke-jr** [comment]
@sipa

**Comment by sipa** [comment]
Unsure. The "(mod n)" at the end just means the entire operation is done modulo n. It's not an operation.

**Comment by dcousens** [comment]
We actually had this come up recently in https://github.com/bitcoinjs/bitcoinjs-lib/issues/929#issuecomment-345909669

**Comment by gspasov** [comment]
@sipa 
Well I didn't know that. And this made me stay on this problem some days until I understood that I wasn't doing the operations in the correct order. I'm sure I wasn't the only one that got confused there and I won't be the last one. After I figured it out I asked some people what did they think this operation is done and they had the same first thought as I did. So either we are all blind or this isn't written the best way possible.

**Comment by MeshCollider** [comment]
The (mod n) at the end is standard in maths, I don't think changing it here improves clarity

**Comment by bpivcard26** [comment]
`marged`

**Comment by melodeys** [comment]
Learning

**Comment by melodeys** [comment]
Keeping fingers crossed

**Comment by melodeys** [comment]
Can hope this works

**Comment by melodeys** [comment]
Keep all updated

**Comment by melodeys** [comment]
Hope getting close

**Comment by melodeys** [comment]
Time will only tell..

**Comment by melodeys** [comment]
Not sure what to do next

**Comment by melodeys** [comment]
Trying to go with the flow uggg..

File: bip-0032.mediawiki [hunk 1]
```diff
@@ -77,7 +77,7 @@ The function CKDpriv((k<sub>par</sub>, c<sub>par</sub>), i) &rarr; (k<sub>i</sub
 ** If so (hardened child): let I = HMAC-SHA512(Key = c<sub>par</sub>, Data = 0x00 || ser<sub>256</sub>(k<sub>par</sub>) || ser<sub>32</sub>(i)). (Note: The 0x00 pads the private key to make it 33 bytes long.)
 ** If not (normal child): let I = HMAC-SHA512(Key = c<sub>par</sub>, Data = ser<sub>P</sub>(point(k<sub>par</sub>)) || ser<sub>32</sub>(i)).
 * Split I into two 32-byte sequences, I<sub>L</sub> and I<sub>R</sub>.
-* The returned child key k<sub>i</sub> is parse<sub>256</sub>(I<sub>L</sub>) + k<sub>par</sub> (mod n).
```
Comment (melodeys) [inline bip-0032.mediawiki]: Not sure what this all means

Comment (melodeys) [reply bip-0032.mediawiki]: Learning I hope anyway

**Pull request closed without merging**